docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -133,40 +117,34 @@ spec:
+** If you use a single node deployment to test the process, replace `worker` with `master` in the `MachineConfig`.
+** Only load the `nvme-fabrics` module because it loads the transport-specific modules, such as TCP, RDMA, or FC, as needed.
 +
 ifeval::["{build}" != "downstream"]
 For production deployments that use NVMe-TCP volumes, it is recommended that you use multipathing. For NVMe-TCP volumes {OpenShiftShort} uses native multipathing, called
 https://nvmexpress.org/faq-items/what-is-ana-nvme-multipathing/[ANA].
 endif::[]
 ifeval::["{build}" != "upstream"]
-For production deployments that use NVMe-TCP volumes, it is recommended that you use multipathing. For NVMe-TCP volumes, {OpenShiftShort} uses native multipathing, called ANA.
+** For production deployments that use NVMe-TCP volumes, use multipathing. For NVMe-TCP volumes, {OpenShiftShort} uses native multipathing, called ANA.
 endif::[]
-+
-After the {OpenShiftShort} nodes reboot and are loading the `nvme-fabrics` module, you can confirm that the operating system is configured and supports ANA by checking the host:
+** After the {OpenShiftShort} nodes reboot and load the `nvme-fabrics` module, you can confirm that the operating system is configured and that it supports ANA by checking the host:
 +
 ----
 $ cat /sys/module/nvme_core/parameters/multipath
 ----
 +
 [IMPORTANT]
-ANA does not use the Linux Multipathing Device Mapper, but {OpenShiftShort} requires `multipathd` on Compute nodes to be running for the {compute_service_first_ref} to be able to use multipathing. Multipathing is automatically configured on data plane nodes when they are provisioned.
-
-Multipathing::
-Multipathing is recommended for iSCSI and FC protocols. To configure multipathing on these protocols, you perform the following tasks:
+ANA does not use the Linux Multipathing Device Mapper, but {OpenShiftShort} requires `multipathd` to run on Compute nodes for the {compute_service_first_ref} to be able to use multipathing. Multipathing is automatically configured on data plane nodes when they are provisioned.
 
-* Prepare the {OpenShiftShort} hosts
-* Configure the Block Storage services
-* Prepare the {compute_service} nodes
-* Configure the {compute_service}
-+
-To prepare the {OpenShiftShort} hosts, ensure that the Linux Multipath Device Mapper is configured and running on the {OpenShiftShort} hosts by using `MachineConfig`. For example:
+Multipathing:::
+** Multipathing is recommended for iSCSI and FC protocols. To configure multipathing on these protocols, you perform the following tasks:
+*** Prepare the {OpenShiftShort} hosts
+*** Configure the Block Storage services
+*** Prepare the {compute_service} nodes
+*** Configure the {compute_service}
+** To prepare the {OpenShiftShort} hosts, ensure that the Linux Multipath Device Mapper is configured and running on the {OpenShiftShort} hosts by using `MachineConfig`. For example:
 +
 [source,yaml]
 ----
```

Comment:
In the following code, can we remove the comments that have a hashtag? The comments are rendering in the doc preview, so customers will be able to see them.

Reply:
The comments were meant to be seen by customers. The 3 comments explain:
- What the file is doing (2 things)
- What the "weird" `mode: 384` actually means for the file permissions, as people are not used to see this value as a decimal number but as an octal number instead.
- What the format of the data in `source` is and how they can use a different way of providing the contents.